MessengerOnFileForLinux/DisplayChat/src/ConsoleWindow.cpp

```diff
@@ -26,34 +26,62 @@ void ConsoleWindow::displayRegistrationWindow()
     move(0,0);
     printw("Hello ");
     printw(getenv("USER"));
-    printw(". You are using messenger first time. You need create account.");
+    printw(". You are using messenger first time. You need create account. Enter your password twice.");
     move(2,0);
+    printw("Enter the password : ");
     refresh();
 }
 
-void ConsoleWindow::displayLoggedWindow()
+void ConsoleWindow::displaySignInWindow()
 {
     clear();
     move(0,0);
     printw("Hello ");
     printw(getenv("USER"));
     printw(". If you want use messenger, you need to log in.");
     move(2,0);
+    printw("Enter the password : ");
     refresh();
 }
 
 std::string ConsoleWindow::getStringFromConsoleWindow()
 {
-    std::string input;
-    nocbreak();
-    echo();
-    int ch = getch();
-    while ( ch != '\n' )
+    std::string output;
+    char enteredCharacter{};
+    while (enteredCharacter != '\n')
     {
-        input.push_back( ch );
-        ch = getch();
+        constexpr int firstAsciiCharacter = 0;
+        constexpr int lastAsciiCharacter = 127;
+        constexpr int delInAsciiCode = 127;
```

Comment:
albo wszędzie Character albo wszędzie Code.

Dodatkowo skoro jedna i druga = 127 to po co dwie zmienne?

Reply:
1. Ok.
2. Chce pokazać że to chodzi o klawisz DEL. Tam jest obłsuga tego co się ma dziać jak nacisnisz delete. Jakby tam było w środku lastAsciiCharacter to byłoby to moim zdaniem mało zrozuimałe

Reply:
To wywal first i last i zrób range na tym i masz problem rozwiązany by nie było dwóch o tym samym numerze.

Reply:
jak?

Reply:
sprawdź w necie jak zrobić ifa pomiędzy dwoma intami, ewentualnie tak jak mówiłeś w funkcję to, po prostu niech to zniknie stąd, bo jest niepotrzebne

Reply:
ok, to już jest w funkcji. Wpadnie w kolejnym commicie po poprawkach do review